src/searchForm/stepper/stepperFullScreen/StepperFullScreen.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Stepper, { StepperProps, StepperDisplay } from 'stepper'
+import Divider from 'divider'
+import Section from 'layout/section/baseSection'
+import ChevronIcon from 'icon/chevronIcon'
+import Item from '_utils/item'
+
+export interface StepperFullScreenProps extends StepperProps {
+  readonly itemTitle: string
+  readonly onClick?: (event: React.MouseEvent<HTMLElement>) => void
+}
+
+export const StepperFullScreen = ({
```

Comment:
Consider renaming StepperFullScreen/DatepickerFullScreen => StepperSection and DatepickerSection.
Can be done in a future PR for both at the same time.

Reply:
We had few discussions about naming so we'll regroup everything in a future PR ;)